test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperPartialExtensionsTest.cs

```diff
@@ -582,5 +618,52 @@ public override string ToString()
                 return "test-model-content";
             }
         }
+
+        internal class TestHtmlHelper : HtmlHelper
```

Comment:
This is where I saw it https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Rendering/HtmlHelperSelectTest.cs#L1575